Automattic/woocommerce-payments
# Use the new Deposit Status Pill on the Deposits table

Fixes #5893 

#### Changes proposed in this Pull Request









This is a small PR that updates the **Payments > Deposits** table to render the new deposit status pill instead of plain text for the deposit status:

![image](https://user-images.githubusercontent.com/2275145/229661310-31e0476d-e5ab-4988-bdcc-488e9f0eee0d.png)

#### Testing instructions





1. Checkout this branch and run `npm run build:client`
2. From the WP Admin page visit **Payments > Deposits**
3. Confirm the new status pill is being used.

-------------------


**Post merge**
